lib/lab_manager/app/endpoints/status.rb

```diff
@@ -0,0 +1,22 @@
+require 'lab_manager/app/endpoints/base'
+require 'json'
+
+module LabManager
+  class App
+    module Endpoints
+      # class responsible for provider statuses
+      class Status < Base
+        get '/v_sphere' do
+          provider_name = 'v_sphere'
+          {
+            max_running_machines: ::Provider::VSphereConfig.scheduler.max_vm,
+            running_machines: ::Compute.where(provider_name: provider_name, state: 'running').count,
+            errored_machines: ::Compute.where(provider_name: provider_name, state: 'errored').count,
+            created_machines: ::Compute.where(provider_name: provider_name, state: 'created').count,
+            queued_machines: ::Compute.where(provider_name: provider_name, state: 'queued').count
```

Comment:
I suggest to pass come possibility to specify what states should be returned; these hardcoded: running, errored, created and queued might be default option. The other thing: v_sphere specific measure is only max_running_machines: the rest is the same for all possible providers :) what about this?